Dockerfile

```
@@ -10,7 +10,7 @@ WORKDIR /app
 
 COPY Gemfile* ./
 
-RUN gem install bundler -v 2.4.22
+RUN gem install bundler -v 2.6.2
```

Version can just be dropped here

```suggestion
RUN gem install bundler
```

---

ACK